### ISSUE TITLE
[KotH/20v20] Sky Push Royale

### ISSUE THREAD

**Issue by arcadeboss** [OvercastCommunity/CommunityMaps#733]
### Checklist

- [X] I have [pruned](https://pgm.dev/docs/guides/packaging/pruning-chunks) the map.
- [X] I have agreed with assigning the CC BY-SA license to this map, as mentioned in the [README](https://github.com/OvercastCommunity/CommunityMaps#readme).
- [X] I have read and understood the  [map submission guidelines](https://oc.tc/submissions).
- [X] I have created an XML file.
- [X] I have created a map image.
- [X] I have uploaded the map zip file to a file sharing service.
- [X] The map has been tested locally to make sure it works.

### Map Name

Sky Push Royale

### Gamemode & Map Description

- Cross between wool-style sky-bridge contests and control-the-point games.
- Kits, supplies, layout are in wool-style, but CTP at sky-level to win.

Set in a medieval tournament context, Red and Blue teams spawn in their castle and use supplied kit and supply chest blocks to build stairs, ladders, etc towards the 5 control points at sky level.   Teams can also take advantage of other resources in their castle including:
- Infinite Iron Block supply
- Regeneration Splash Spawner
- Portal to Sky Platform (above team's home control point) for Archer attacks on areas below.  Jump to return.
- Fire bows/arrows stored on the castle roof, accessed by a series of ladders.  (Limited by arrows)
- Dungeon with a few extra items like flint and steel, Knockbones (knockback bone), upgrades, etc.

There are 5 control points: Red, Center, Blue, Orange and Yellow.   Center is worth 3, Red/Blue worth 2 and Orange/Yellow worth 1 point per second.   Point control expires slowly.    Teams fight for 2000 points to win, games should last 10 to 15 minutes.

The three main control points also have their control status displayed in the platform above them.   This allows the control status to be seen from afar.  The two smaller points have Web spawners.

Players can build anywhere in the central void, allowing many options on sky access and griefing, etc.

To keep sky level more interesting the following may be placed: Webs, Water, Fire, TNT, pressure plates, buttons.

Gaps for kill, TNT double-kill reward.

### Team Sizes

20v20 (Should work from teams of 5 to 45)

### Screenshots

https://imgur.com/a/8R3oy1k

### XML

https://gist.github.com/arcadeboss/86c0b27cc225668d57e67f8ef8c120bc

### Map Image

![map](https://github.com/OvercastCommunity/CommunityMaps/assets/3164203/be5e2ea7-7c45-4f96-abf2-d149821f0787)

### Map Download

https://www.mediafire.com/file/l6p22jvxxft8qsx/sky-push-royale-zip.zip/file

**Comment by eliminate123** [comment]
This looks cool, cant wait to play! 👍

**Comment by TheRealPear** [comment]
This is an interesting concept. However my only complaint are the broadcasts in the first few minutes of the game- it might be too spammy for the chat. It would be best to combine some of them (l54-56) into one tip message.

**Comment by arcadeboss** [comment]
Yeah, I didn't prefer to break it up, but it's a lot of information that gets lost in a single message.   I certainly tried it first, but it didn't work.

Also it is critical game information, and actually dinging the bell a few times actually helps to get the attention of players.

But, if there is a way to present the same information on multiple lines within a single alert, that would be a reasonable compromise, if not superior.  Please let me know if that exists.

**Comment by arcadeboss** [comment]
@TheRealPear - I couldn't find any way to do this in a single alert.

I'd prefer not to change the messages as they have been most meticulously crafted.

But..... if you are saying that the tips to players are deal breaker, I will change it.   

Please confirm.

**Comment by calcastor** [comment]
I feel like the messages aren't too intrusive; they aren't sent all at once and in-game chat will break it up. The audio used for the relevant chat broadcasts is also pretty non-intrusive.

**Comment by arcadeboss** [comment]
@calcastor - Thanks.  Pear said here that it was okay on the messages: https://discord.com/channels/86514356862320640/752217058112176308/1122214795614109797
